### PR TITLE
Zilla Plus reference docs for kafka-proxy binding and aws vault

### PR DESCRIPTION
**Pull request by jfallows**

### CODE REVIEW

File: src/reference/config/bindings/binding-kafka-proxy.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  type: kafka-proxy
+  kind: proxy
+  options:
+    external:
+      host: kafka-#.external.net
+      port: 9093
+    internal:
+      host: b-#.kafka.internal.net
+      port: 9094
+  exit: tls_client0
```
Comment (vordimous) [inline src/reference/config/bindings/binding-kafka-proxy.md:26]: I don't think the trialing 0s are needed anymore?

File: src/reference/config/bindings/binding-kafka-proxy.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+:::: note Properties
+
+- [kind\*](#kind)
+- [options](#options)
+- [options.external](#options-external)
+  - [external\[\].host](#external-host)
+  - [external\[\].port](#external-port)
+- [options.internal](#options-internal)
+  - [internal\[\].host](#internal-host)
+  - [internal\[\].port](#internal-port)
```
Comment (vordimous) [inline src/reference/config/bindings/binding-kafka-proxy.md:44]: if options.external and options.internal are just objects and not lists of objects then we don't need the square brackets `\[\]`

File: src/reference/config/vaults/vault-aws.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+#### overrides.resourcegroupstaggingapi
+
+> `string`
+
+Endpoint URL override for AWS Resource Groups Tagging API.
+
+---
+
+::: right
+\* required
```
Comment (vordimous) [inline src/reference/config/vaults/vault-aws.md:95]: Are there no required properties for this vault?

Comment (jfallows) [reply src/reference/config/vaults/vault-aws.md:95]: Right, the only options are to override AWS service URL endpoints, but the defaults are typically used in most cases, so no properties are required.

**Review by vordimous — APPROVED** [review]

**Pull request merged**